version.c

```diff
@@ -419,18 +426,19 @@ void print_version(FILE *fp)
   fprintf(fp, "\nGPGme: %s", mutt_gpgme_print_version());
 #endif
 
+#ifdef HAVE_NOTMUCH
+  fprintf(fp, "\nlibnotmuch: %d.%d.%d", LIBNOTMUCH_MAJOR_VERSION,
```

Comment:
I'd spell this as "notmuch", as that seems the spelling used on https://notmuchmail.org/

Reply:
I wasn't sure what to put as the two names have different version numbers:

- `notmuch -v` : 'notmuch 0.29.1'
- `ls /usr/lib...` : 'libnotmuch.so.5.2.0'

NeoMutt only sees the '5.2.0'

Reply:
Oh, it's the shared library's version, not the pacakge version.. confusing.. I'd prefer notmuch to dump the package version, which is what users look at generally

Reply:
.. but notmuch doesn't (provide a package version), so yeah, better than nothing